### PR TITLE
adding support for laravel 5.7.* in composer.json

### PR DESCRIPTION
**Pull request by CryDeTaan**
Not sure if this will cause other issues but unable to use with latest version of Laravel.

### CODE REVIEW

**Comment by CryDeTaan** [comment]
All checkes passed, support laravel 5.7

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     ],
     "require": {
         "php": ">=5.6.0",
-        "illuminate/support": "5.1.*|5.2.*|5.3.*|5.4.*|5.5.*|5.6.*",
+        "illuminate/support": "5.1.*|5.2.*|5.3.*|5.4.*|5.5.*|5.6.*|5.6.*",
```
Comment (denpamusic) [inline composer.json]: Why are you using 5.6.* twice here? For laravel 5.7 it should be `"illuminate/support": "5.1.*|5.2.*|5.3.*|5.4.*|5.5.*|5.6.*|5.7.*"`

**Comment by CryDeTaan** [comment]
This commit should be ignored. did not make the necessary change.

File: composer.json (hunk 1 above)
Comment (CryDeTaan) [reply composer.json]: I just realised my error.

Comment (CryDeTaan) [reply composer.json]: I will submit a new PR.

Comment (denpamusic) [reply composer.json]: OK. Thank you!

Comment (CryDeTaan) [reply composer.json]: Apologies, about that. Will be creating a new Laravel 5.7.2 project, will let you know if I find any other issues.

**Pull request closed without merging**